scripts/remap-submodules-to-https.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+# Copyright 2019-present 650 Industries. All rights reserved.
+
+set -e
+
+sed -i.bak 's#git@github.com:#https://github.com/#g' $1 >> $1
```

Comment:
```suggestion
sed -i.bak 's#git@github.com:#https://github.com/#g' $1
```

sed will overwrite the file with these options, also i don't think we want to append to the file

Reply:
I tried suggestion, it didn't work